### DISCUSSION TITLE
AWX vs semaphore

### DISCUSSION THREAD

**Discussion by dberardo-com** [semaphoreui/semaphore#960] [General]
What are advantages of Semaphore when compared against AWX?

**Comment by feloxx** [comment]
semaphore have the following advantages :

- lightweight
- contracted
- low learning cost

**Reply by fiftin** [reply]
Many users love Semaphore for API, as I know :)

Pros:
- no docker needed
- modern responsive UI
- written in pure Go - cross-compilation, no deps, single binary file per platform
- ready to use after installation  `snap install semaphore`

**Reply by dberardo-com** [reply]
thanks for the answer, which authentication types are supported/planned for M2M API ? 

access tokens vs. Oauth2, LDAP, etc. ?

**Comment by dberardo-com** [comment]
alright, is there an official roadmap of which features are planned next?

**Reply by fiftin** [reply]
https://github.com/ansible-semaphore/semaphore/projects?type=classic

**Reply by fiftin** [reply]
Currently I focused on bugfixing.

**Comment by tboerger** [comment]
I'm closing this as the question seem to be answered :)
